server/db/schema.sql

```diff
@@ -1,9 +1,18 @@
-  DROP TABLE IF EXISTS pet_types;
-
-  CREATE TABLE pet_types(
-    id SERIAL PRIMARY KEY,
-    name VARCHAR (255) NOT NULL,
-    img_url VARCHAR (255) NOT NULL,
-    description TEXT
-  );
-
+DROP TABLE IF EXISTS pet_types,
+pets CASCADE;

```

Comment:
Nice! Can we add some newlines between each statement here -- after lines 2 and 8?